src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -283,3 +283,76 @@ function update(
 )
   return update(Algorithm(alg), bpc; kwargs...)
 end
+
+function rescale_message(bp_cache::AbstractBeliefPropagationCache, partitionedge)
```

Comment:
I still find it confusing that this is called `rescale_message` but it rescales multiple messages, one in each direction along the partition edge. An alternative could be:
```julia
function rescale_messages(bp_cache::AbstractBeliefPropagationCache, partitionedge::PartitionEdge)
```

Reply:
Yeah that's a good point. I have done that rename.